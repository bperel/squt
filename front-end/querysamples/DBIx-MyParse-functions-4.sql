SELECT expr LIKE '%test_' ESCAPE '|'
