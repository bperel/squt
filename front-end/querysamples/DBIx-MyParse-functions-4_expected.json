{
   "Warning" : {
      "No alias" : {
         "like" : ["function", "SELECT"]
      }
   },
   "Tables" : {
      "" : {
         "" : {
            "OUTPUT" : {
               "expr" : {
                  "0" : "expr"
               }
            }
         }
      }
   },
   "Functions" : {
      "0" : {
         "to" : "OUTPUT",
         "Constants" : {
            "|" : "|",
            "%test_" : "%test_"
         },
         "name" : "like",
         "alias" : "0"
      }
   }
}
